core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2305,6 +2305,23 @@ private[spark] object Utils extends Logging {
     log.info(s"Started daemon with process name: ${Utils.getProcessName()}")
     SignalUtils.registerLogger(log)
   }
+
+  /**
+  * Unions two comma-separated lists of files.
+  */
+  def setUnionFileLists(leftList: String, rightList: String): Set[String] = {
```

Comment:
This is a weird name. How about just `unionFileLists`? You could also use varargs for the arguments instead of just allowing two lists, but that's minor.